mlocal/checks/basechecks.chk

```diff
@@ -62,13 +62,11 @@ fi
 if [ $do_go_version_check = 0 ]; then
 	if [ "$hostgo" = "" ]; then
 		printf " checking: looking for Go compiler... "
-		for go in $hstgo_opts; do
-			gopth=`command -v $go`
-			if [ "$gopth" != "" ]; then
-				hstgo=$gopth
-				break
-			fi
-		done
+        gopth=`command -v $go`
+        if [ "$gopth" != "" ]; then
+            hstgo=$gopth
+            break
+        fi
```

Comment:
This has an indentation problem.  This script is using all tabs the way go requires it, so please make sure your editor isn't putting in spaces instead, so it stays consistent.